sound/soc/sof/core.c

```diff
@@ -337,6 +337,22 @@ static int sof_probe_continue(struct snd_sof_dev *sdev)
 	return ret;
 }
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_PROBE_WORK_QUEUE)
+static void sof_probe_work(struct work_struct *work)
+{
+	struct snd_sof_dev *sdev =
+		container_of(work, struct snd_sof_dev, probe_work);
+	int ret;
+
+	dev_info(sdev->dev, "%s entry\n", __func__);
```

Comment:
dev_dbg()?

Reply:
Ah yes. For some reason I was convinced info was lower priority than debug, I stand corrected.